src/components/30-organisms/modal/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.3
+
+Fix: hide close button, if `forced` is true.
```

Comment:
Open a github issue describing why this was requested, possibly linked there to JIRA. Then you can follow our tradition of referencing the github issue after your sentence :-)